project_stock/tests/test_project_stock.py

```
@@ -78,6 +78,20 @@ def test_project_task_without_analytic_account(self):
         self.task.action_done()
         self.assertFalse(self.task.stock_analytic_line_ids)
 
+    def test_project_task_picking_done_analytic_items(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_project_task_picking_done_analytic_items`

Add return type annotation: `None`

(ANN201)